apollo-backend/src/test/java/io/logz/apollo/AuthTest.java

```diff
@@ -79,6 +79,38 @@ public void testLogin() throws Exception {
         apolloTestClient.login();
     }
 
+    @Test
+    public void testEnabled() throws Exception {
+        ApolloTestAdminClient apolloTestAdminClient = standaloneApollo.createTestAdminClient();
+        ApolloTestClient apolloTestClient = standaloneApollo.createTestClient();
+
+        User testUser = apolloTestClient.getTestUser();
+
+        // Login admin and signup user
+        apolloTestAdminClient.login();
+        apolloTestAdminClient.signup(testUser, Common.DEFAULT_PASSWORD);
+
+        apolloTestClient.login();
+
+        // Check that its valid
+        assertThat(apolloTestClient.getAllServices()).size().isGreaterThanOrEqualTo(0);
+
+        testUser.setEnabled(false);
+        apolloTestAdminClient.updateUser(testUser);
+
+        // Check that user is forbidden even without logging in
+        assertThatThrownBy(apolloTestClient::getAllUsers).isInstanceOf(ApolloNotAuthorizedException.class);
```

Comment:
Why is the request here not also getAllServices?
Why not choose a lighter request?

Reply:
good catch